enos/enos-scenario-autopilot.hcl

```diff
@@ -25,6 +29,12 @@ scenario "autopilot" {
       arch    = ["arm64"]
       edition = ["ent.fips1402", "ent.hsm", "ent.hsm.fips1402"]
     }
+
+    # PKCS#11 can only be used on ent.hsm and ent.hsm.fips1402.
+    exclude {
+      seal    = ["pkcs11"]
```

Comment:
Do we need to consider architecture in these excludes?

Reply:
It's already happening up above. There are not ent.fips1402, ent.hsm, or ent.hsm.fips1402 editions for arm64.